### DISCUSSION TITLE
راهنمای اضافه کردن فالبک به نسخه جدید مرزبان

### DISCUSSION THREAD

**Discussion by s43id** [Gozargah/Marzban#306] [Q&A]
از دوستانی که بلدن این کارو انجام بدن
ممنون میشم بگن چطوری فالبک رو به نسخه جدید اضافه کنیم

**Comment by omid-sadeghi** [comment]
این [ویدئوی یوتیوب](https://www.youtube.com/watch?v=CX8bXLD69G0&t=1638s) خیلی خوب توضیح داده.
داخل [گیت‌هاب](https://github.com/XTLS/Xray-examples/blob/main/All-in-One-fallbacks-Nginx/nginx.conf) هم یک کانفیک کامل برای فالبک بعلاوه‌ی تنظیمات nginx را گذاشته.
همین دو تا لینک بالا کفایت میکنه. اما بازم اطلاعات بیشتر خواستی به سایت [پروژه‌ی xray](https://xtls.github.io/en/) مراجعه کن. بطور مفصل توضیح داده، لینکاشو پایین میذارم:
[لینک 1](https://xtls.github.io/en/config/features/fallback.html)
[لینک 2](https://xtls.github.io/en/document/level-1/fallbacks-lv1.html)
[لینک 3](https://xtls.github.io/en/document/level-1/fallbacks-with-sni.html)
[لینک 4](https://xtls.github.io/en/document/level-0/) (این لینک یه آموزش مبتدیانه از راه‌اندازی xray هست از صفر که قسمت هفتمش فالبک رو توی فایل config.json بکار برده و در کل آموزش جالبیه)
